book/website/history.md

```diff
@@ -0,0 +1,40 @@
+# History of The Turing Way project
+
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
+Founded by Dr. Kirstie Whitaker, the project began as a Book of Reproducible Research in 2018, created by Dr. Whitaker and close allies and partners: assisted in creation by working with members of the REG team at the Turing.
+The project was incubated during Kirstie's time as a [Mozilla Open Leader](https://mozilla.github.io/leadership-training/), and was funded by a grant within the Turing.
+
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A github repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (i.e. Collaboration Cafe, Book Dash).
+In-person workshops were given in person on tools aligned with the project: particulary Binder and JupterBook, and the project was presented in many conferences across the UK and internationally.
+
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019, The Turing Way expanded into five guides.
+As a co-founder of Open Life Science (which uses many Mozilla OL frameworks), Malvika helped to incubate what are now key partnerships for The Turing Way.
+This period corresponded with the COVID-19 pandemic lockdowns, which transitioned many processes of contribution, collaboration, and documentation into remote ones.
+
+In part due to this, The Turing Way was able to expand greatly, from one guide on Reproducibility into five different guides related to project design, communication, collaboration, and research ethics.
+A global (and multilingual) community began to engage with The Turing Way, and the process of translation/localisation began informally.
+A Community Handbook was also established to document best practices that could be used by other communities.
+The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
+The project received a few awards & citations in the 2021 London Mayor report, HiddenREF Awards, and Goldacre Report.
+The Book Dash Planning Committe was established to plan the community's flagship collaboration event.
```

Comment:
when?

Reply:
in 2022 - adding this for clarity, good catch @jcolomb!